.github/workflows/deploy_cms.yaml

```diff
@@ -39,12 +39,11 @@ jobs:
     name: Build and push CMS image
     runs-on: ubuntu-latest
     continue-on-error: false
-    environment: ${{ inputs.environment }}
-    # These permissions are needed to interact with GitHub's OIDC Token endpoint.
-    permissions:
-      id-token: write
-      contents: read
-
+    strategy:
+      matrix:
+        environment: [ 'prod' ]
+    environment: ${{ matrix.environment }}
+    
```

Comment:
It seems that the [input](https://github.com/pagopa/b2b-portals/pull/119/files#diff-1122a44dbe5ba245e5894a343a89cca77ccadb3a04a1d9f6096df7078ca618bbR12) variable is no longer being used, correct?

Reply:
Correct, I remove it

Reply:
I have removed it in the last commit